### PR TITLE
Lims 4281/4297

### PR DESCRIPTION
**Pull request by dianaeldemerdash**

### CODE REVIEW

File: ui_testing/testcases/basic_tests/test03_orders.py [hunk 1]
```diff
@@ -487,3 +488,69 @@ def test013_update_order_number(self):
...
+
+        self.order_page.get_orders_page()
+        self.order_page.get_random_order()
+        order_url = self.base_selenium.get_url()
+        self.base_selenium.LOGGER.info(' + order_url : {}'.format(order_url))
+        self.order_page.get_material_type()
+        self.order_page.set_material_type(material_type=test_plan_dict['Material Type'])
+        self.order_page.confirm_popup(force=True)
+        self.order_page.set_article(article_name=test_plan_dict['Article Name'])
+        self.order_page.get_article()
```
Comment (0xIslamTaha) [inline ui_testing/testcases/basic_tests/test03_orders.py]: there is no need to get_article here as its value should be test_plan_dict['Article Name']

File: ui_testing/testcases/basic_tests/test03_orders.py [hunk 2]
```diff
@@ -487,3 +488,69 @@ def test013_update_order_number(self):
...
+        self.order_page.set_test_plan(test_plan=test_plan_dict['Test Plan Name'])
+        self.order_page.get_test_plan()
```
Comment (0xIslamTaha) [inline ui_testing/testcases/basic_tests/test03_orders.py]: there is no need to get_test_plan here as its value should be test_plan_dict['Test Plan Name']

File: ui_testing/testcases/basic_tests/test03_orders.py [hunk 3]
```diff
@@ -487,3 +488,69 @@ def test013_update_order_number(self):
...
+        self.order_page.get_suborder_table()
+        if 'save_btn' == save:
+            self.order_page.save(save_btn='order:save_btn')
+        else:
+            self.order_page.cancel(force=True, cancel_btn='order:cancel_btn')
+
+        self.base_selenium.get(url=order_url, sleep=5)
```
Comment (0xIslamTaha) [inline ui_testing/testcases/basic_tests/test03_orders.py]: where is the assertion?

File: ui_testing/pages/order_page.py [hunk 4]
```diff
@@ -31,18 +31,38 @@ def get_material_type(self):
...
-                element='order:article', item_text=article)
+
+    #def set_article(self, article=''):
+        #if article:
+            #self.base_selenium.select_item_from_drop_down(element='order:article', item_text=article)
+        #else:
+            #self.base_selenium.select_item_from_drop_down(element='order:article')
+            #return self.get_article()
+
+    def set_article(self, article_name='', article=''):
```
[the remark this answers is not in the document]
Comment (dianaeldemerdash) [reply ui_testing/pages/order_page.py]: @0xIslamTaha In this line self.order_page.set_article(article_name=test_plan_dict['Article Name']) when I give to it article it didn't work so I added article_name to the set method

**Pull request merged**